source/Halibut.Tests/Support/Certificates.cs

```diff
@@ -36,7 +41,13 @@ static Certificates()
             TentaclePollingPfxPath = Path.Combine(directory, "TentaclePolling.pfx");
             TentaclePolling = new X509Certificate2(TentaclePollingPfxPath);
             TentaclePollingPublicThumbprint = TentaclePolling.Thumbprint;
-            Ssl = new X509Certificate2(Path.Combine(directory, "Ssl.pfx"), "password");
+            
+            WrongPfxPath = Path.Combine(directory, "WrongCert.pfx");
```

Comment:
I kept the wrong cert, since I do want the thumbprint to be valid and so it is easy for anyone in the future to see where the thumbprint is coming from.